src/common/config/state/runtime.js

```diff
@@ -6,7 +6,9 @@ import { BUILD_ENV, DIST_METHOD, VERSION } from '../../constants/env'
 
 const model = {
   buildEnv: BUILD_ENV,
-  bytesSent: {},
+  bytesSent: {}, // Used for SM to capture body bytes sent per endpoint
+  queryBytesSent: {}, // Used for SM to capture query parameter bytes sent per endpoint
+  imgMethodCount: 0, // Used for SM to capture count of time img network method is used
```

Comment:
I'm debating if we should keep storing things like this in the runtime.

If we send imgMethodCount = 2 on page hide, then the page is restored, imgMethodCount = might become 4 and send, skewing the average.

It might be safer/better to just invoke the SM subscriber directly, which will unload at page hide time and reset its state -- making each pagehide/pageshow an independent count.... Lets discuss